Functions.Templates/Templates-v2/EventHubTrigger-Python/template.json

```diff
@@ -0,0 +1,228 @@
+                "getConnnectionString",
+                "readFileContent_FunctionApp",
+                "replaceTokensInText_FunctionApp",
+                "writeFile_FunctionApp",
+                "showMarkdownPreview"
+            ]
+        },
+        {
+            "name": "Add EventHubTrigger function to the main file",
+            "actions": [
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_FunctionBody",
+                "replaceTokensInText_FunctionBody",
+                "appendFileContent_FunctionApp"
+            ]
+        },
+        {
+            "name": "Create New Blueprint file",
+            "input" : {
+                "userCommand" : "createBluePrint",
+                "assignTo" : "$(BLUEPRINT_FILENAME)"
+            },
+            "actions": [
+                "getBluePrintFileName",
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_BlueprintFile",
+                "replaceTokensInText_BlueprintFile",
+                "readFileContent_BlueprintBody",
+                "replaceTokensInText_BlueprintBody",
+                "writeFile_BlueprintFile",
+                "appendFileContent_BlueprintBody",
+                "readFileContent_BlueprintImport",
+                "replaceTokensInText_BlueprintImport",
+                "writeFile_BlueprintImport",
+                "appendFileContent_FunctionApp",
+                "readFileContent_BlueprintRegister",
+                "replaceTokensInText_BlueprintRegister"
+            ]
+        },
+        {
+            "name": "Add EventHubTrigger function to the Blueprint",
+            "actions": [
+                "getBluePrintFileName",
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_BlueprintBody",
+                "replaceTokensInText_BlueprintBody",
+                "appendFileContent_BlueprintBody"
+            ]
+        }
+    ],
+    "actions" : [
+        {
+            "name": "getFunctionName",
+            "type": "UserInput",
+            "assignTo": "$(FUNCTION_NAME_INPUT)",
+            "paramId": "trigger-functionName",
+            "defaultValue": "EventhubTrigger"
+        },
+        {
+            "name": "getEventHubName",
+            "type": "UserInput",
+            "assignTo": "$(EVENTHUB_NAME)",
+            "paramId": "trigger-eventhubName",
+            "defaultValue": "MyEventHub"
+        },
+        {
+            "name": "getConnnectionString",
+            "type": "UserInput",
+            "assignTo": "$(EVENTHUB_CONNECTION_STRING)",
+            "paramId": "trigger-connectionStringName",
+            "defaultValue": ""
+        },
+        {
+            "name": "readFileContent_FunctionApp",
+            "type": "ReadFromFile",

```

Comment:
I have a few concerns with the `ReadFromFile` action:
1. I would assume `WriteToFile` and `ReadFromFile` are using the same root directory, but write is using the app directory and read is getting the file from the template itself
2. At least in the old schema, the template file is stored directly in the generated templates.json, so tooling is never actually "reading" anything

I say we get rid of `ReadFromFile` for now and only add it back if we have scenarios in the future where we need to read a _user's_ file. For actions like `WriteToFile`, I think the "source" property should be able to directly reference a template file name instead of needing to use `$(FUNCTION_APP_CONTENT)` as an intermediary
```
"source": "function_app.py"
```